IPython/core/completerlib.py

```diff
@@ -155,7 +156,7 @@ def is_importable(module, attr, only_modules):
 
 def try_import(mod, only_modules=False):
     try:
-        m = __import__(mod)
+        m = import_module(mod)
     except:
         return []
     mods = mod.split('.')
```

Comment:
There's some code here that can be cleaned up now. When you `__import__('foo.bar')`, it actually returns the `foo` module, and you have to get `bar` manually. `import_module` does this for us.

```
 1│ __import__('IPython.display')
 1> <module 'IPython' from '/home/takluyver/.local/lib/python3.5/site-packages/IPython/__init__.py'>

 2│ import importlib

 3│ importlib.import_module('IPython.display')
 3> <module 'IPython.display' from '/home/takluyver/.local/lib/python3.5/site-packages/IPython/display.py'>
```